addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -215,3 +215,18 @@ def test_import_partner_peppol_fields(self):
         # The partner should be retrieved based on the peppol fields
         imported_invoice = self.import_attachment(xml_attachment, self.company_data["default_journal_sale"])
         self.assertEqual(imported_invoice.partner_id, partner)
+
+    def test_import_partner_postal_address(self):
+        " Test importing postal address when creating new partner from UBL xml."
+        file_path = "bis3_bill_example.xml"
+        file_path = f"{self.test_module}/tests/test_files/{file_path}"
+        with file_open(file_path, 'rb') as file:
+            xml_attachment = self.env['ir.attachment'].create({
+                'mimetype': 'application/xml',
+                'name': 'test_invoice.xml',
+                'raw': file.read(),
+            })
+
+        bill = self.import_attachment(xml_attachment)
+
+        self.assertEqual(bill.partner_id.contact_address_complete, "270 rte d'Arlon, 8010 Strassen, Luxembourg")
```

Comment:
Great addition thanks ! Maybe just show the partner address is not XXX before you import your XML and it will be perfect :D